prereise/gather/griddata/hifld/data_process/generators.py

```diff
@@ -0,0 +1,136 @@
+import pandas as pd
+from powersimdata.utility.distance import haversine
+
+from prereise.gather.griddata.hifld import const
+from prereise.gather.griddata.hifld.data_access import load
+
+
+def floatify(value, default=float("nan")):
+    """Return a float if possible, otherwise return a default value.
+
+    :param object value: value to be coerced to float, if possible.
+    :param object default: value to be returned if float conversion isn't possible.
+    :return: (*float/object*) -- float or default value as applicable.
```

Comment:
I think the returned type is `float` now, no?

Reply:
Not if the user supplies an alternate default value that's not a float.

Reply:
Yeah, indeed